test/parser.test.coffee

```diff
@@ -156,11 +156,30 @@ module.exports =
     equ r.sample.$$[10].$$[2]._, '3'
     equ r.sample.$$[10].$$[3]['#name'], 'one'
     equ r.sample.$$[10].$$[3]._, '4'
-    equ r.sample.$$[10].$$[4]['#name'], 'two'
+    equ r.sample.$$[10].$$[4]['#name'], 'two' 
```

Comment:
Ugh. I must have fat-fingered that line when I was copy-pasting it into a new test. I cut my finger earlier this week, so I'm going to blame the annoying bandage on my finger. 

My apologies.